src/home_robot_hw/home_robot_hw/remote/modules/manip.py

```diff
@@ -331,11 +331,5 @@ def _compute_base_translation_pos(self):
         return (l0_pose.inverse() * l1_pose).translation()[0]
 
     def _extract_joint_pos(self, q):
-        return [
-            q[HelloStretchIdx.BASE_X],
-            q[HelloStretchIdx.LIFT],
-            q[HelloStretchIdx.ARM],
-            q[HelloStretchIdx.WRIST_YAW],
-            q[HelloStretchIdx.WRIST_PITCH],
-            q[HelloStretchIdx.WRIST_ROLL],
-        ]
+        """Helper to convert to just what we're using in this class."""
```

Comment:
Couldnt understand what the docstring meant.

Reply:
fixed